### ISSUE TITLE
Use Storybook to show our presentational components

### ISSUE THREAD

**Issue by pcowgill** [tasitlabs/tasit#32]
This will help us to identify which components are general enough that they belong in a `tasit-components` dir at the top level of this repo (that is, outside of the demo app). That way they can be used in other apps in this repo or used by the `tasit-cli` when generating an app scaffold for developers using the `tasit-sdk`.

It will also be nice to link to the storybook from the developers page on the website for the project.

https://storybook.js.org/
https://storybook.js.org/basics/guide-react-native/

**Comment by pcowgill** [comment]
@nirmalb9 This issue in particular would be a good one for someone interested in front-end / mobile

**Comment by mseijas** [comment]
@pcowgill I'm giving this a shot. Thoughts on changing the folder structure a bit so that we have one folder per component?

This way we can keep all of these together, say:

``` 
  /Button
     /index.js (exports Button)
     /Button.js
     /Button.test.js
     /Button.story.js
```

**Comment by pcowgill** [comment]
@mseijas Thanks!! That works for me. @marcelomorgado any objections?

**Comment by marcelomorgado** [comment]
No, no objections. I've created an issue for that: https://github.com/tasitlabs/tasit/issues/98
Is this change have to be done only for presentational components?

**Comment by pcowgill** [comment]
Hmm, good question. Let's just do it with presentational components to start. We can always do it for the container components / screens in a later PR if we find that that's helpful too.

**Comment by pcowgill** [comment]
I responded about presentational components vs. container components over in the other issue,
